packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin-conf

```diff
@@ -11,3 +11,11 @@ flags:
     passDeviceSpecs: {{default true settings.kubernetes.device-plugins.nvidia.pass-device-specs}}
     deviceListStrategy: {{default "volume-mounts" settings.kubernetes.device-plugins.nvidia.device-list-strategy}}
     deviceIDStrategy: {{default "index" settings.kubernetes.device-plugins.nvidia.device-id-strategy}}
+{{#if (gt settings.kubernetes.nvidia.device-plugins.max-sharing-per-gpu 1)}}
```

Comment:
Does the `gt` operator work as expected if the value is not set at all? Or does this need to be guarded by another conditional?